src/SignalR/clients/java/signalr/signalr.client.java.javaproj

```diff
@@ -11,6 +11,8 @@
 
     <IsShippingPackage>true</IsShippingPackage>
 
+    <IsTestProject>true</IsTestProject>
```

Comment:
Isn't this project more like a sample?

Reply:
No, this is the java library we ship, and also the project that we run the tests from.